### DISCUSSION TITLE
MOCK HSS Container is taking restart itself inside FEG.

### DISCUSSION THREAD

**Discussion by Aniket621** [magma/magma#15285] [Communications]
Hi all,
We are deploying the Magma FEG and Mock HSS FEG build successfully and all container are showing running but facing issue while run the Mock HSS container. Container create successsfully but it's taking restart itself. As per container logs getting error while fetch the configuration from this file /var/opt/magma/configs/gateway.mconfig.
Container Logs below  please help regarding this error.
I0814 09:58:56.700706       1 service_config.go:161] Successfully loaded '::service_registry' service configs from '/etc/magma/service_registry.yml'
I0814 09:58:56.701526       1 service_config.go:161] Successfully loaded '::service_registry' service configs from '/etc/magma/service_registry.yml'
I0814 09:58:56.771872       1 service_config.go:161] Successfully loaded 'feg::hss' service configs from '/etc/magma/feg/hss.yml'
I0814 09:58:56.772273       1 service.go:125] Registering Service303 as a public GRPC server.
I0814 09:58:56.772806       1 service_config.go:110] Successfully loaded structured '::magmad' service configs from '/etc/magma/magmad.yml'
E0814 09:58:56.775398       1 config.go:78] hss Managed Configs Load Error: No configs found for service: 'hss' in /var/opt/magma/configs/gateway.mconfig
2023/08/14 09:58:56 Error getting hss config: No configs found for service: 'hss' in /var/opt/magma/configs/gateway.mconfig
2023/08/14 09:58:56 Error validating config: missing port in address

**Comment by Aniket621** [comment]
Hi all, Now this above issue is resolved.

**Comment by panyogesh** [comment]
Closing in the issue.
